### ISSUE TITLE
Multiple SDKs not supported on Dotnet SDK installed as Snap

### ISSUE THREAD

**Issue by shakeelansari63** [dotnet/sdk#34593]
### Describe the bug
I have installed Dotnet SDK from Snap store via latest beta channel and it is installed correctly. But there is no way to install and older version of SDK inside same installation. This restrict me from developing on projects which need older SDKs. 

So now I have Dotnet SDK 8 with Dotnet Runtime 8. 
But I also have some projects which use SDK 7, SDK 6 and SDK 5.

There is no way to install all these SDKs alongside SDK 8 which is already installed. 

Though Dotnet Support having multiple SDKs alongside, the Snap version does not have this feature. 

### To Reproduce

Install Dotnet from Snap Store - 
```
sudo snap install dotnet-sdk --beta --classic
```

Check the SDKs from installed dotnet
```
dotnet --list-sdks
```

And it will give only SDK. 

### Further technical details
- Include the output of `dotnet --info`
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version

**Comment by md-owes** [comment]
I do have a similar issue and it would be inconsistent if i had stable dotnet v7 installed via snap and preview version v8 installed via apt or manual.

**Comment by soumyaDghosh** [comment]
You can have parallel installs of snaps. Though, it's an experimental features, but atleast it worked for me.

```bash
 soumyaxubuntu  soumyas-GF63-Thin-11UC  ~  which dotnet
/snap/bin/dotnet

 soumyaxubuntu  soumyas-GF63-Thin-11UC  ~  dotnet --version
8.0.204

 soumyaxubuntu  soumyas-GF63-Thin-11UC  ~  dotnet-sdk_7.dotnet --version
7.0.408
```

https://snapcraft.io/blog/parallel-installs-test-and-run-multiple-instances-of-snaps

**Comment by shakeelansari63** [comment]
Yes that is possible. But in a scenario, if my solution has multiple projects and  some projects use a different SDK version than others. This will not work. 

This may be a niche case, but I have seen this scenario.

**Comment by marcpopMSFT** [comment]
Old issue triage: This is not supported by us, you'd have to report this to the snap maintainers if you require this. Note that you don't need to use an older SDK to build when targeting older runtimes. You should only ever need one sdk on the box in the absence of specific global.json pinning requirements.
